app/dao/fact_billing_dao.py

```diff
@@ -108,9 +108,20 @@ def fetch_sms_billing_for_all_services(start_date, end_date):
     return query.all()
 
 
-def fetch_nightly_billing_counts(process_day):
+def fetch_nightly_billing_counts(process_day: date):
+    """
+    This function can also take a datetime instance because
+    isinstance(<datetime instance>, date) is True.
+    """
+
+    # If process_day is a datetime instance, datetime.combine ignores the arguments after the first.
     start_date = convert_local_timezone_to_utc(datetime.combine(process_day, time.min))
     end_date = convert_local_timezone_to_utc(datetime.combine(process_day + timedelta(days=1), time.min))
+
+    print("fetch process_day =", process_day)  # TODO
```

Comment:
These lines should be removed I'm guessing? Since they are just print statements and have a TODO at the end.

Reply:
Yes, they should be removed.